### PR TITLE
Maintenance refresh for server bring-up

### PR DESCRIPTION
**Pull request by jayaddison**
When following through the [server README instructions](https://github.com/PGPerfFarm/pgperffarm_server/blob/9388665391dac38469974ca8a3cc9061c865e1f9/README.server.md) I ran into a few issues; this pull request aims to address them:

- It looks like `pycryptodomex` v3.4.7 (hm, not present in the [repo's tag list](https://github.com/Legrandin/pycryptodome/tags), odd) has compatibility issues with Python 3.x due to number-type suffixes ([this one](https://github.com/Legrandin/pycryptodome/commit/8c4824982bb10218504a0ce60c9b8a3191d92a19#diff-e2ea8fdebf9f9eaa7339bc0f7146b817e5efc751791ad6042cba90c374349e87L399) seemed to be the first line to complain)
- Generation of initial migrations doesn't automatically occur for every registered Django application, so despite the fact that server's apps [are registered in `settings.INSTALLED_APPS`](https://github.com/PGPerfFarm/pgperffarm_server/blob/9388665391dac38469974ca8a3cc9061c865e1f9/rest_api/rest_api/settings.py#L41-L47), it's necessary to list them when running `python3 manage.py makemigrations`

Maybe controversially I've committed the generated migrations to the branch; happy to undo that if the preferred workflow for the project is to generate them on-demand.

The second issue was basically the same as #27, I think (it happened when attempting to load Django fixture data).

### CODE REVIEW

**Comment by jayaddison** [comment]
:wave: cc @ila @rulyox for potential review

**Comment by ila** [comment]
hey, sorry for some reason I missed this - thanks for bringing that up! this will be reviewed in the upcoming days :)

**Comment by jayaddison** [comment]
No problem :)  thank you!

File: rest_api/requirements.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ django==3.2.13
 requests==2.25.1
 urllib3==1.26.5
 psycopg2==2.8.5
-pycryptodomex==3.4.7
+pycryptodomex==3.14.1
```
Comment (YedilSerzhan) [inline rest_api/requirements.txt:5]: Hi! The versions of the package are said to be in line with [pgweb](https://github.com/postgres/pgweb). According to the requirements of pgweb, the [pycryptodomex version](https://github.com/postgres/pgweb/blob/a83b51923a5da51a60f7df48c3742c0f23457ae3/requirements.txt#L3) should be >=3.4.7,<3.5.  @ila Could this be a problem?

Comment (jayaddison) [reply rest_api/requirements.txt:5]: I didn't realize that, thanks @YedilSerzhan - based on that, I'd prefer to maintain dependency consistency (including versions) if we can, even if it maybe isn't crucial in a technical sense.

I've started a related thread in the `pgsql-www` mailing list [here](https://www.postgresql.org/message-id/CALDQ5NzR6wL8LzmAyDTsX2P0WaxZ9XxiMEWMQZ6x6yqcWt%2BEEA%40mail.gmail.com) to see if there's any appetite for upgrading the dependency version there first.

**Pull request closed without merging**